### ISSUE TITLE
Library Statistics graph/info (For newly Added Movies/Series)

### ISSUE THREAD

**Issue by StarSmasher44** [Tautulli/Tautulli#1403]
TL;DR VERSION: I want a counter to show me how many series & movies I added over the last X days, you used to have this feature in a not-so-easily readable graph, this would be a better version.
I just want to know how much movies I added/removed this week, mainly added.

**Is your feature request related to a problem? Please describe.**
Not as much as a problem, I believe this used to be a function in Tautulli, where I could bring up a graph to show me how the amount of movies/series/music has risen in whatever amount of time. And while it wasn't great (Too little info), it worked okay for me to judge how much my library had grown over the past week/weeks after adding and removing things.

**Describe the solution you'd like**
I'd like a way to see the X amount of movies/libraries that I have added in X days or weeks, much like the current system displays stats for X amount of days, I just want to see the amount of movies I had before and after adding and removing a bunch of stuff.

**Describe alternatives you've considered**
None, I like tautulli and like to continue using it, this would just give us a little bit more statistical information, and for me to keep track of how much content I add or remove.

**Additional context**
I guess the easiest way would be to just check for the time it was added, and if that is within X days of your filter it will add to the number.
